internal/provider/cluster_resource.go

```diff
@@ -1363,6 +1379,16 @@ func loadClusterToTerraformState(
 			PrivateNetworkVisibility: types.BoolValue(clusterObj.GetNetworkVisibility() == client.NETWORKVISIBILITYTYPE_PRIVATE),
 			CidrRange:                types.StringValue(clusterObj.CidrRange),
 		}
+		// As noted in the comment of this function, plan may be nil if we
+		// are reading a datasource or importing a resource. In these cases,
+		// since there is currently no indication from the SDK whether a
+		// cluster supports PCR or not, we leave the field blank.
+		if plan != nil && plan.DedicatedConfig != nil {
+			// Otherwise, if we have explicitly specified this flag in the
+			// plan, then we add it to the state. If it was null, we keep it
+			// as null.
+			state.DedicatedConfig.SupportPhysicalClusterReplication = plan.DedicatedConfig.SupportPhysicalClusterReplication
```

Comment:
I'm wondering if it's fine that we don't return the value back from the cluster.  It means for instance that if someone were to import a cluster with this property set on it, it wouldn't be verifiable.  Do you know what happens in this case?